README.md

```diff
@@ -1 +1,84 @@
-Working in progress.
+
+# Table of Contents
+
+- [Installation](#installation)
+  * [Install k2](#install-k2)
+  * [Install lhotse](#install-lhotse)
+  * [Install icefall](#install-icefall)
+- [Run recipes](#run-recipes)
+
+## Installation
+
+`icefall` depends on [k2][k2] for FSA operations and [lhotse][lhotse] for
+data preparations. To use `icefall`, you have to install its dependencies first.
+The following subsections describe how to setup the environment.
+
+CAUTION: There are various ways to setup the environment. What we describe
+here is just one alternative.
+
+### Install k2
+
+Please refer to [k2's installation documentation][k2-install] to install k2.
+If you have any issues about installing k2, please open an issue at
+<https://github.com/k2-fsa/k2/issues>.
+
```

Comment:
It will be more friendly to put conda or pip installation guide here rather than build from source code.

Reply:
Thanks. I just remove this kind of information and refer the reader to the documentation of k2 for installation instructions.